src/components/header/index.tsx

```diff
@@ -1,15 +1,36 @@
+import Image from 'next/image';
+import ButtonSetting from '@/components/buttonSetting';
+import Filter from '@/components/filter';
 import ThemeSetting from '../themeSetting';
 
 function Header(): JSX.Element {
```

Comment:
add a doc or explanation for this component so others can understand the workings of this .